dbt_project.yml

```
@@ -1,15 +1,16 @@
 config-version: 2
 name: 'quickbooks'
  
-version: '0.6.0'
+version: '0.6.1'
```

To be consistent with the other changes in this release
```suggestion
version: '0.7.0'
```

---

Applied!